e2e/tests/omnibus-base-sepolia-andromeda.toml/Perps_Trading.e2e.js

```diff
@@ -258,14 +258,14 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
     const marketId = 200;
     const settlementStrategyId =
       require('../../deployments/extras.json').btc_pyth_settlement_strategy;
-    const maxSize = await PerpsMarketProxy.getMaxMarketSize(marketId);
-    log({ marketId, maxSize });
+    const { maxMarketSize } = await PerpsMarketProxy.getMaxMarketSize(marketId);
```

Comment:
Why did not we add another method `PerpsMarketProxy.getMaxMarketValue(marketId)` instead and did not make the change **non breaking**?